libraries/AP_Airspeed/AP_Airspeed_I2C.cpp

```diff
@@ -54,9 +68,8 @@ bool AP_Airspeed_I2C::init()
 void AP_Airspeed_I2C::_measure()
 {
     _measurement_started_ms = 0;
-    if (hal.i2c->writeRegisters(I2C_ADDRESS_MS4525DO, 0, 0, NULL) == 0) {
-        _measurement_started_ms = AP_HAL::millis();
-    }
+    _dev->write_register(0, 0);
```

Comment:
I don't think this works. The call will just return an error due to 0 length for both read and write buffers. What I ended up doing on my review to change on this driver was to implement a smbus_quick() method. See https://github.com/lucasdemarchi/ardupilot/commits/review-4222 

I couldn't test though. Would be nice to know if it works.